### ISSUE TITLE
[Feature request] Keep refreshing image when not moving cursor

### ISSUE THREAD

**Issue by KaKi87** [stuartlangridge/magnus#25]
Thanks

**Comment by stuartlangridge** [comment]
You can run magnus from the command line (or a shortcut) with the `--force-refresh` parameter which will ensure that the image is refreshed even if the cursor doesn't move, which should solve this!

**Comment by KaKi87** [comment]
Hello,

I just tried `magnus --force-refresh` from a terminal but it doesn't change anything.

Thanks

**Comment by stuartlangridge** [comment]
Oh. Um... it ought to! Can you say what you're running Magnus on? OS details and so on? It should work...

**Comment by KaKi87** [comment]
Distro : Ubuntu 23.04
Desktop environment : Unity
App version : 1.0.3

**Comment by stuartlangridge** [comment]
OK, so, just to confirm, this is the experience I get: if I run as `magnus`, there's no automatic updates, and if I run as `magnus --force-refresh`, updates do happen, like this:

https://github.com/stuartlangridge/magnus/assets/4356350/51f27d23-189f-45c7-8798-6444ed61f1c8

**Comment by KaKi87** [comment]
Yes, this is what I hoped to experience, but I'm not experiencing it.

**Comment by stuartlangridge** [comment]
OK; I wanted to confirm that we were talking about the same thing. I'm afraid I do not know what might be causing that. I'll leave this issue open and update it if I can work something out. Apologies for this; it's supposed to work.

**Comment by qingbyin** [comment]
Hello, I have encountered the same issue, using the same runtime environment and magnus version. Adding --force-refresh had no effect. Has there been any progress on a fix?

**Comment by luigifab** [comment]
Same here, Magnus 1.0.3-4 with Debian Testing, with `magnus --force-refresh`.

![Image](https://github.com/user-attachments/assets/4358116d-b06a-4c72-9e0a-b75cd42581fe)

**Comment by luigifab** [comment]
Strangely, when I copy paste the source code from #30, it's working!
